packages/vscode/package.json

```diff
@@ -96,7 +96,7 @@
     "@types/mocha": "^10.0.1",
     "@types/vscode": "^1.68.1",
     "@vscode/test-electron": "2.2.0",
-    "@vscode/vsce": "^2.19.0",
+    "@vscode/vsce": "^2.21.1",
```

Comment:
this was also upgraded, because when I was building the extension, the output told me to do upgrade

Reply:
Please revert this, since it has nothing to do with the stated purpose of the PR

Reply:
done